src/cloudWatchLogs/registry/logDataRegistry.ts

```diff
@@ -241,7 +233,7 @@ export async function filterLogEventsFromUri(
     }
 
     const msgId = uriToKey(createURIFromArgs(logGroupInfo, parameters))
-    const msgTimeout = await Message.putMessage(msgId, `${logGroupInfo.groupName}`, {
+    const msgTimeout = await Messages.putMessage(msgId, `Loading from: '${logGroupInfo.groupName}'`, {
         message: logGroupInfo.streamName ?? '',
     })
```

Comment:
@nkomonen-amazon is this one intentionally different? I'm seeing it popup separately. And has different wording.